themes/bootstrap3/templates/Recommend/ConsortialVuFind.phtml

```diff
@@ -0,0 +1,31 @@
+<div class="consortialVuFind">
+  <?php $results = $this->recommend->getResults() ?>
+  <?php $records = $results['records'] ?? [] ?>
+  <?php $resultCount = $results['resultCount'] ?? 0; ?>
+  <?php $moreResultsUrl = $this->recommend->getMoreResultsUrl(); ?>
+  <?php if (!empty($records)): ?>
+    <h2><?=$this->transEsc('consortial_vufind_recommend_heading')?></h2>
+    <?php if ($this->translate('consortial_vufind_recommend_intro', [], '')): ?>
+      <p><?=$this->translate('consortial_vufind_recommend_intro', [
+          '%%result_count%%' => $this->localizedNumber($resultCount),
+          '%%url%%' => $moreResultsUrl,
+      ])?></p>
+    <?php endif; ?>
+    <div class="list-group">
+      <?php foreach ($records as $record): ?>
+        <div class="list-group-item">
+          <span>
+            <a href="<?=$record['url']?>" class="title icon-link" target="_blank">
+              <?=$this->icon('external-link', 'icon-link__icon') ?>
+              <span class="icon-link__label"><?=$this->escapeHtml($record['title'])?></span>
+            </a>
+          </span>
+        </div>
+      <?php endforeach; ?>
+      <a class="list-group-item" href="<?=$moreResultsUrl?>">
+        <?=$this->icon('external-link', 'icon-link__icon') ?>
+        <?=$this->transEsc('More results')?>...
```

Comment:
There is no "More results" translation in the language files, and we've also moved away from putting ellipses outside of the translation strings. If you want this, I'd suggest adding `more_results_ellipses` as a translation string, and using the actual unicode ellipses character in the translation string instead of `...`.

If this was copied and pasted from somewhere else, maybe we need to fix something somewhere... but I couldn't find an obvious source for it.

Reply:
Yes thank you, implemented.  This was an accidental copy from our local environment where we do have a 'More results' key.